<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryWidget</class>
 <widget class="QWidget" name="GeometryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>74</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bt_load">
         <property name="toolTip">
          <string>Load/Set Detector Geometry</string>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_apply">
         <property name="toolTip">
          <string>Assemble Data</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_save">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Save geometry</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Detector:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_detectors"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Geometry File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_geometry_file"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Click &quot;Apply&quot; to draw image; Click on quadrant to select then CTRL+arrow-keys to move them.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
